test/unit/text_direction_test.rb

```diff
@@ -5,7 +5,7 @@ class TextDirectionTest < ActiveSupport::TestCase
 
   paths.each do |path|
     test "explicitly sets text direction for #{File.basename(path)}" do
-      data = YAML.load(File.read(path))
+      data = YAML.load(File.read(path)) # rubocop:disable Security/YAMLLoad
```

Comment:
I think it'd be useful to note in the commit, or in a comment why we allow this one to not go through safe_load, as it's not super clear to me why it's ok here.

Reply:
I've added more detail to the commit message and also raised https://github.com/alphagov/whitehall/issues/3684